src/routes/safe/components/Balances/Tokens/screens/AssetsList/index.tsx

```diff
@@ -38,7 +37,7 @@
~      asset.symbol.toLowerCase().includes(filter.toLowerCase()),
```

Comment:
what happens with loading status? Is it ok remove it completely? I think it should be handled when nftAssetsList it's undefined, then show the message if the return value is `[]`